nuvla/job_engine/connector/helm_driver.py

```diff
@@ -81,9 +84,10 @@ def run_command(self, command: list) -> CompletedProcess:
                '--kubeconfig', self.k8s.kubeconfig()] \
             + command
 
-        log.debug('Helm command to run %s ', cmd)
+        if log.level == logging.DEBUG:
```

Comment:
Do we need this if? Depending on log configured on the command line the logger should filter debug logs if level is higher than debug

Reply:
Because after that I have `' '.join(cmd)`, which will always be executed even if the mode is DEBUG. Hence, I guarded it by simple boolean comparision.

Reply:
OK. I see this is raising the eyebrows, so I will remove this.